pylint/checkers/strings.py

```diff
@@ -912,12 +919,51 @@ def process_non_raw_string_token(
             # character can never be the start of a new backslash escape.
             index += 2
 
+    @check_messages("possible-forgotten-f-prefix")
     @check_messages("redundant-u-string-prefix")
     def visit_const(self, node: astroid.Const):
         if node.pytype() == "builtins.str" and not isinstance(
             node.parent, astroid.JoinedStr
         ):
-            self._detect_u_string_prefix(node)
+            self._detect_possible_f_string(node)
+        self._detect_u_string_prefix(node)
+
+    def _detect_possible_f_string(self, node: astroid.Const):
+        """Check whether strings include local/global variables in '{}'
+        Those should probably be f-strings's
+        """
+
+        def detect_if_used_in_format(node: astroid.Const) -> bool:
```

Comment:
Why did you remove this function? I think it worked quite okay in my previous commits?

Reply:
I tried to simply detect a format call without checking assign_name but failed. (This is really just a work in progress it can be reverted)